setup.cfg

```diff
@@ -29,7 +29,7 @@ packages = find:
 install_requires =
     audbackend >=0.3.17
     audeer >=1.18.0
-    audformat >=0.16.0,<2.0.0
+    audformat >=0.16.1,<2.0.0
```

Comment:
Is there a specific reason we require `<2.0.0` for `audformat`?

Reply:
Not really. I guess it's a valid assumption that this does will not work, but for other packages we also no longer limit it like this.
I would say it makes only sense if there would be already a version 2.0.0 available or on the horizon, so I guess we can remove this (which I would do in a different pull request).

Reply:
Good, then let's remove it in a separate PR.

Reply:
I created https://github.com/audeering/audb/pull/265